src/v2/components/AddBlock/index.tsx

```diff
@@ -223,10 +224,10 @@ class AddBlock extends PureComponent<AddBlockProps> {
           </Container>
         )}
       </DropZoneUploader>
-    );
+    )
   }
 }
 
-export default graphql<AddBlockProps>(createBlockMutation, {
+export default graphql(createBlockMutation, {
```

Comment:
Try 
```
export default graphql<Props>(createBlocMutation, { ... })(AddBlock)
```

Reply:
Ah you know what there's two different sets of props here; pre-hoc and post-hoc.